docs/book/src/plugins/go-v2-plugin.md

```diff
@@ -0,0 +1,22 @@
+# go/v2 plugin
+
+This plugin is also used to build operators similar to [go/v3](go-v3-plugin.md). Check the following [example](https://github.com/kubernetes-sigs/kubebuilder/tree/master/testdata/project-v2) to view the files created or updated by this plugin when used with the supported subcommands.
+
+## How to use it ?
```

Comment:
```suggestion
## When should I use this plugin

Only if you are looking to scaffold a project with the legacy layout. Otherwise, it is recommended you to use the default Golang version plugin. 

<aside class="note warning">

Be aware that this plugin version does not provide a scaffold compatible with the latest versions of the dependencies used in order to keep its backwards compatibility. 

</aside>


## How to use this plugin ?
```